ts/src/coincatch.ts

```diff
@@ -360,56 +360,18 @@ export default class coincatch extends Exchange {
                     'CRO': 'CronosChain',
                 },
                 'networksById': {
-                    'BITCOIN': 'BTC',
```

Comment:
these were just inverted duplicates, whcih are already being auto-generated. however, i've left some of them which had "todo check" flags, idk RayBastard might need to revise them